sdk/keyvault/azure-keyvault-certificates/samples/backup_restore_operations.py

```diff
@@ -6,16 +6,16 @@
 import time
 from azure.keyvault.certificates import CertificateClient, CertificatePolicy
 from azure.identity import DefaultAzureCredential
-from azure.core.exceptions import HttpResponseError
 
 # ----------------------------------------------------------------------------------------------------------
 # Prerequisites:
 # 1. An Azure Key Vault (https://docs.microsoft.com/en-us/azure/key-vault/quick-create-cli)
 #
 # 2. azure-keyvault-certificates and azure-identity packages (pip install these)
 #
-# 3. Set Environment variables AZURE_CLIENT_ID, AZURE_TENANT_ID, AZURE_CLIENT_SECRET, VAULT_URL
-#    (See https://github.com/Azure/azure-sdk-for-python/tree/main/sdk/keyvault/azure-keyvault-keys#authenticate-the-client)
+# 3. Set up your environment to use azure-identity's DefaultAzureCredential. To authenticate a service principal with
```

Comment:
We are trying to not direct customers to using client credentials for authentication, you might want to rephrase to something like [this](https://github.com/Azure/azure-sdk-for-java/blob/8b31a1f08916c8b8089aa756710f1ebc75e98c8d/sdk/keyvault/azure-security-keyvault-certificates/src/samples/java/com/azure/security/keyvault/certificates/BackupAndRestoreOperations.java#L42). There's a bunch of files in the Certificates samples doing the same.